articles/terraform/terraform-vm-msi.md

```diff
@@ -0,0 +1,92 @@
+# Terraform with a Linux Viratul Machine on Azure
```

Comment:
You need to add the article metadata to the top of this article.

Reply:
Added.